### PR TITLE
Mcq content

### PR DESCRIPTION
**Pull request by AdeshkumarNaik**

### CODE REVIEW

**Review by mangesh-sakordekar — APPROVED** [review]

File: src/MDX/tutorials/react/props.mdx [hunk 1]
```diff
@@ -6,24 +6,28 @@ next: /tutorials/react/state
 prev: /tutorials/react/components
 ---
 
+import MCQ from "components/mcq";
```
Comment (dustinc555) [inline src/MDX/tutorials/react/props.mdx:9]: None html components should be typically capitalized.

File: src/templates/mdx_template.js [hunk 2]
```diff
@@ -11,12 +11,16 @@ import SEO from "components/seo";
 import Content from "components/ContentBanner";
 import HyperLink from "components/highLightLink";
 import PrevNextNav from "components/prevNextNav";
+// eslint-disable-next-line no-unused-vars
+import MCQ from "components/mcq";
+//import Truefalse from "components/truefalse";
```
Comment (dustinc555) [inline src/templates/mdx_template.js:16]: I don't know if commented out imports are really serving a purpose here

File: src/components/truefalse.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+
+class Truefalse extends Component {
```
Comment (dustinc555) [inline src/components/truefalse.js:3]: Typical naming convention here would be TrueFalse, cap per word.

File: src/components/threechoicemcq.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+
+class Threechoicemcq extends Component {
```
Comment (dustinc555) [inline src/components/threechoicemcq.js:3]: The casing here bugs me a little bit too, why does it have to be a three choice mcq, why not an N choice?

**Review by dustinc555 — CHANGES_REQUESTED** [review]

File: src/templates/mdx_template.js (hunk 2 above)
Comment (AdeshkumarNaik) [reply src/templates/mdx_template.js:16]: They are not I shall remove them

File: src/components/truefalse.js (hunk 3 above)
Comment (AdeshkumarNaik) [reply src/components/truefalse.js:3]: truefalse?

File: src/components/threechoicemcq.js (hunk 4 above)
Comment (AdeshkumarNaik) [reply src/components/threechoicemcq.js:3]: cuz it's threechoices

Comment (AdeshkumarNaik) [reply src/components/threechoicemcq.js:3]: It's fine over here.

**Review by dustinc555 — APPROVED** [review]

**Pull request merged**